src/core/Akka/Actor/ActorSystem.cs

```diff
@@ -74,6 +74,24 @@ public abstract class ActorSystem : IActorRefFactory, IDisposable
         public abstract ILoggingAdapter Log { get; }
 
         /// <summary>
+        /// Start-up time in milliseconds since the epoch.
+        /// </summary>
+        public long StartTime => _startTime;
+
+        private readonly long _startTime = MillisecondsSinceEpoch();
+
+        /// <summary>
+        /// Up-time of this actor system in seconds.
+        /// </summary>
+        public long Uptime => (MillisecondsSinceEpoch() - StartTime) / 1000;
```

Comment:
I have used the same signature as the scala version but would also prefer Timespan.